spells.yml

```diff
@@ -3406,6 +3665,23 @@ fabricate:
     - artificer
     - transmutation
     - wizard
+face_swap:
+  casting_time: 1 standard action
+  components: S
+  concentration: false
+  description: |-
+    Make a melee spell attack against a creature within 5 feet of you that's the same size as you. On a hit, you become disguised as the target as if by _disguise self_ and vice versa.
```

Comment:
Perhaps it'd be worthwhile to add a clause about how this can also be performed on a willing target?